dashboard/frontend/pages/_app.js

```diff
@@ -0,0 +1,20 @@
+import { Provider } from 'mobx-react';
+import { useStore } from '../state/stores';
+
+import '../styles/index.scss';
+import { mountSockets } from '../actions';
+
+let socket;
+
+export default function App({ Component, pageProps }) {
+    const store = useStore(pageProps.initialState)
+    socket = mountSockets(store);
+
+    return (
+        <Provider store={store}>
```

Comment:
Didn't we decide to move away from the `Provider` way of adding stores? We can simply import the store instead, no?